### ISSUE TITLE
Selectors do not insert a null row when used on an empty table

### ISSUE THREAD

**Issue by jsternberg** [influxdata/flux#3064]
The selector functions min, max, first, and last do not insert a null row when used on an empty table. This results in a discrepancy when used with `aggregateWindow` since they don't insert a row. That means when `window(every: inf)` is used the empty windows are then removed.

We should consider a way to make it so the selectors operate similarly to aggregates for `aggregateWindow` because otherwise it's not possible to fill in empty values when using these functions.

As a workaround, the `sum()` function can be used with a selector to get this behavior.

```
aggregateWindow(every: 5m, fn: (tables=<-, column) => tables |> min(column) |> sum(column))
```

This will only work for numeric types. I am not aware of a workaround for string or boolean types.

**Comment by nathanielc** [comment]
As a heads up, adding a null row makes sense in the context of an aggregate but it does not make sense in the context of a selector. I am not sure how we should handle that difference but we want to make sure we do not regress that behavior for selectors with what ever solution we find.

**Comment by jsternberg** [comment]
Closed in favor of #10561.
